app/assets/stylesheets/admin.scss

```
@@ -147,6 +147,10 @@ label {
   gap: var(--su-4);
   padding-top: var(--su-6);
   padding-bottom: var(--su-2);
+
+  .warning-padding {
+    padding: var(--su-2);
+  }
```

```suggestion
  .auth-warning {
    padding: var(--su-2); // Override default crayons notice padding
  }
```

```suggestion
  .auth-warning {
```

I think this is a bit more future-proof rather than include `-padding` in the css class name. When I see the `-padding` here, I almost think it's something from crayons or even a utility class, so I think that could potentially be confusing. But `auth-warning` is more clear that this is a custom class that's overriding the default padding [that comes from crayons](https://github.com/forem/forem/blob/master/app/assets/stylesheets/components/notices.scss#L4) (which is why I'm suggesting the additional comment here 😄)